mypy/options.py

```diff
@@ -74,6 +74,7 @@ class BuildType:
 ) - {"debug_cache"}
 
 _based = True
+_legacy = False
```

Comment:
isnt this redundant if theres already a `_based` variable

Reply:
the `--strict` flags and stuff need different options, all to get the upstream tests to pass expectadly.

This stuff is a little messy, maybe I can look to refactor it at some point.